spec/collections/deltaloader/delta_load_registry.yaml

```diff
@@ -5,7 +5,7 @@ schema:
   "$schema": http://json-schema.org/draft-07/schema#
   title: delta_load_registry
   type: object
-  description: Don't touch this. It's for the exlusive use of delta loaders.
+  description: Don't touch this. It's for the exclusive use of delta loaders.
```

Comment:
I swear this was corrected in a previous PR.....

Reply:
it was in a PR on `relation_engine_spec` which I'm migrating over to this repo